packages/constants/src/__fixtures__/boolean.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2024 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+// biome-ignore lint/suspicious/noExplicitAny: <explanation>
+type Callable = (...a: any[]) => any;
+type Fixture = [string, boolean, Callable, Values][];
+// biome-ignore lint/suspicious/noExplicitAny: <explanation>
+type Values = any[];
+
+export const truthy: Values = [
+  1,
+  '1',
+  'on',
+  'true',
+  'yes',
+  true,
+  'ON',
+  'YES',
+  'TRUE',
+];
+export const falsey: Values = [
+  0,
+  '0',
+  'off',
+  'false',
+  'no',
+  false,
+  [],
```

Comment:
I don't think that `[]` or `{}` are something that we can safely consider falsey.

It feels like you're trying to suggest that only empty versions of these objects would considered falsey(?)

But I'm pretty sure that if you put a populated version into the truthy values, it would fail.

Reply:
I do not think that they should be considered as either "false" or "true" personally. However, that is what was in the source before I got to it so I was keeping them for consistency only. https://github.com/gohypergiant/standard-toolkit/pull/108/files#diff-49742f138b932b2304849053f633a630876e6a9d469f2fb6cc7d3d9351378024L24-L25

Reply:
I tend to agree that they shouldn't be in either group. It does feel like something that should be fixed

Reply:
I'd be happy to refactor these out if that is the direction we want to go... but in another PR.

Reply:
Yea: https://github.com/gohypergiant/standard-toolkit/pull/108/files#diff-9e7c7e0fa586596636eafa34b75c343bcc48590b97395798c7d78dca3aea8ea1R15 this is problematic since any object will stringify to that value

Reply:
I did note this slightly in the code; https://github.com/gohypergiant/standard-toolkit/pull/108/files#diff-9e7c7e0fa586596636eafa34b75c343bcc48590b97395798c7d78dca3aea8ea1R13-R14.